src/envo/envo-edit.owl

```diff
@@ -6170,27 +6178,24 @@ AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#has
 AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> "ADL:FTT"^^xsd:string) <http://www.geneontology.org/formats/oboInOwl#hasRelatedSynonym> <http://purl.obolibrary.org/obo/ENVO_00000091> "strand"^^xsd:string)
 AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> "USGS:SDTS"^^xsd:string) <http://www.geneontology.org/formats/oboInOwl#hasRelatedSynonym> <http://purl.obolibrary.org/obo/ENVO_00000091> "strand"^^xsd:string)
 AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/ENVO_00000091> "beach"^^xsd:string)
-SubClassOf(<http://purl.obolibrary.org/obo/ENVO_00000091> <http://purl.obolibrary.org/obo/ENVO_00000191>)
+SubClassOf(<http://purl.obolibrary.org/obo/ENVO_00000091> <http://purl.obolibrary.org/obo/ENVO_01001884>)
 SubClassOf(<http://purl.obolibrary.org/obo/ENVO_00000091> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/BFO_0000050> <http://purl.obolibrary.org/obo/ENVO_00000304>))
 SubClassOf(<http://purl.obolibrary.org/obo/ENVO_00000091> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/RO_0002473> ObjectIntersectionOf(<http://purl.obolibrary.org/obo/ENVO_00002007> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/RO_0002473> ObjectUnionOf(<http://purl.obolibrary.org/obo/ENVO_00002139> <http://purl.obolibrary.org/obo/ENVO_00002140> <http://purl.obolibrary.org/obo/ENVO_01000017> <http://purl.obolibrary.org/obo/ENVO_01000018> <http://purl.obolibrary.org/obo/UBERON_0006612>)))))
 
 # Class: <http://purl.obolibrary.org/obo/ENVO_00000092> (sea beach)
 
-AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> "https://en.wikipedia.org/wiki/Beach"^^xsd:string) <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_00000092> "A landform consisting of loose rock particles such as sand, gravel, shingle, pebbles, cobble, or even shell fragments along the shoreline of a sea."^^xsd:string)
+AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> "https://en.wikipedia.org/wiki/Beach"^^xsd:string) <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_00000092> "A beach which is part of a sea shore."^^xsd:string)
```

Comment:
the original definition is much better.   To understand this revised definition, I have to figure out what you mean by 'beach' and what you mean by 'sea shore'....